komorebi/src/window_manager.rs

```diff
@@ -1476,7 +1486,11 @@ impl WindowManager {
 
         let workspace = self.focused_workspace_mut()?;
         let target_idx = match workspace.layout() {
-            Layout::Default(_) => 0,
+            Layout::Default(layout) => match layout {
```

Comment:
Here let's use the same `matches!` path as above to return early instead of engaging with this logic